packages/form-footer/src/FormFooter.story.tsx

```diff
@@ -41,8 +41,13 @@ const meta: StoryMetaType<typeof FormFooter> = {
     generate: {
       storyNames: ['LightMode', 'DarkMode'],
       combineArgs: {
-        backButtonText: [undefined, 'Back'],
-        cancelButtonText: ['', 'Cancel'],
+        backButtonProps: [
+          undefined,
+          { text: 'Back', leftGlyph: undefined },
+          { text: 'Back', leftGlyph: null },
+        ],
```

Comment:
Seemed superfluous to include a generated set of stories showing the ability to use a custom `leftGlyph` since most teams will still use the `ArrowLeftIcon`. Open to adding if we prefer to have greater coverage

Reply:
This is fine. This is just for visual tests, so we just need to know an icon renders correctly